src/theme/light-theme.tsx

```diff
@@ -66,7 +66,7 @@ const shadows = [
     ),
 ] as Theme['shadows']
 
-const spacing = 5
+const spacing = (factor: number = 1) => Math.floor(factor) * 4
```

Comment:
Nit: a matter of taste, but I prefer to have all logic outside the parameters:

```suggestion
const spacing = (factor?: number) => Math.floor(factor ?? 1) * 4
```